modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/resources/com/liferay/dynamic/data/mapping/dependencies/test-structured-content-nested-structure.json

```diff
@@ -0,0 +1,450 @@
+{
+	"availableLanguageIds": [
+		"en_US"
+	],
+	"defaultLanguageId": "en_US",
+	"definitionSchemaVersion": "2.0",
+	"fields": [
+		{
+			"collapsible": false,
+			"ddmStructureId": "",
+			"ddmStructureKey": "",
+			"ddmStructureLayoutId": "",
+			"fieldReference": "Fieldset39644786",
+			"label": {
+				"en_US": "Fields Group"
```

Comment:
Since there are more than one fields group, can you add 1/2/3 in the end?
`"en_US": "Fields Group 1" `

Reply:
and to the rest of JSON file please.

Reply:
Sure!